my-sms/spec/models/title_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Title, type: :model do
+  subject { title }
+  let(:title) { build(:title, title_params) }
+  let(:title_params) { {} }
+
+  context 'with valid params' do
+    it { should validate_presence_of(:name) }
```

Comment:
Nice, like the use of implicit subject here. One nitpick though is that they do prefer the `expect` matcher rather than `should` version, you can achieve the same like

```suggestion
    it { is_expected.to validate_presence_of(:name) }
```

Reply:
Fixed in 9c1858a113f78d27f678467ebe5a7b6fb3efd5ed